weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/usePlaygroundState.ts

```diff
@@ -88,10 +88,26 @@ const NUMERIC_SETTINGS_MAPPING: Record<
   },
 };
 
-export const usePlaygroundState = () => {
-  const [playgroundStates, setPlaygroundStates] = useState<PlaygroundState[]>([
-    DEFAULT_PLAYGROUND_STATE,
-  ]);
+const getDefaultModelState = (model: LLMMaxTokensKey): PlaygroundState => {
```

Comment:
super nit: unify on either `model` or `modelId` below, those seem pretty different but are the same here?